### PR TITLE
Add problem data for Homework 8 Question 4: Online advertising displays

### PR DESCRIPTION
**Pull request by nasheqlbrm**
I added the problem data for Homework 8 Question 4: Online advertising displays. I generated the data from Matlab and have verified from the Convex Optimization course grader that it is correct.

As a reference, the following would be the change in the solution values:
image

image

### CODE REVIEW

File: homework/homework_8.ipynb [hunk 1]
```diff
@@ -1,671 +1,827 @@
...
-    "# Question 4: Online advertising displays"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "### Convert Matlab data generation to numpy\n",
-    "In this case, we're bound to get different answers because we're seeding the RNG, and we've translated it from Matlab,\n",
-    "so the RNG will be different."
```
Comment (ali-tny) [inline homework/homework_8.ipynb:500]: Can you remove this line, now that we're no longer generating random data?

File: homework/homework_8.ipynb [hunk 2]
```diff
@@ -1,671 +1,827 @@
...
+      "source": [
+        "print(f\"Greedy net profit: {revenue.value - penalty.value:.2f}\")\n",
+        "print(f\"Associated greedy revenue: {revenue.value:.2f}\")\n",
+        "print(f\"Associated total penalty: {penalty.value:.2f}\")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        ""
```
Comment (ali-tny) [inline homework/homework_8.ipynb]: please can you remove this final empty cell?

**Review by ali-tny — CHANGES_REQUESTED** [review]
thanks a lot for this contribution!

just two comments to clean up the notebook a little bit (removing a comment about random data that would be misleading now that we're using the same data as Matlab, and just removing an empty cell from the end of the notebook)

ideally, if you could squash the data commits into a single commit and the notebook commits into a single commit (so it's just 2 commits) that would be good (but don't worry too much about it)

File: homework/homework_8.ipynb (hunk 1 above)
Comment (nasheqlbrm) [reply homework/homework_8.ipynb:500]: Done.

**Comment by nasheqlbrm** [comment]
I have made the requested changes. Please let me know if anything else is needed.

**Review by ali-tny — APPROVED** [review]
awesome thanks!

**Pull request merged**